tempel.el

```diff
@@ -580,7 +581,17 @@ This is meant to be a source in `tempel-template-sources'."
   "Template completion is done."
   (interactive)
   ;; TODO disable only the topmost template?
-  (while tempel--active (tempel--disable)))
+  (while tempel--active (tempel--done)))
+
+(defun tempel--done (&optional st)
```

Comment:
Would it make sense to unique tempel--done and tempel--disable in a single function with a DONE argument?

Reply:
Hm, I think I see your point. It's a tough call, imho. In my mind, `disable` is not just a variation of `done`, but more of a sub-task of it. If it was up to me, I think it'd leave it as is, but I don't feel strongly about it.